extensions/redfish-config

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+err_base="extensions/redfish"
+
+# sysfs mountpoint
+opt_rootdir=""
+# relative fw path (unused)
+opt_fw_path=""
+# enable list only action
+opt_action="show-config"
+# load a kernel module
+opt_modprobe=""
+#set -- `getopt r:p: "$@"`
+while [ $# -gt 0 -a "$1" != "--" ]; do
+	opt="$1"; shift;
+	case $opt in
+	-r)	opt_rootdir="$1";		shift;;
+	-p)	opt_fw_path="$1";		shift;;
+	-m)	opt_modprobe="$1";		shift;;
+	-l)	opt_listonly="list-ifnames"	;;
```

Comment:
`opt_listonly` isn't used, lets remove it.

Reply:
We don't need to modprobe and relative path either --> it's for compatibility to the ibft extension.